### DISCUSSION TITLE
Problem saving pkml files from R

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#1878] [Questions & Problems]
Hi all,

I am trying to modify the dose in a pkml file in R and save the file back on the computer.
The issue is that once the file is saved back it does not open and MoBi give an error message.
The simulation itself runs in R, also after modification. 
I was able to reproduce the issue with a dummy model and you have attached the initial pkml file, the script I use to modify the doses and the newely created pkml file not working.

[script.zip](https://github.com/user-attachments/files/18077960/script.zip)

Do you know where the issue come from?

Thanks a lot in advance for your help!

Best,

Donato

**Comment by PavelBal** [comment]
Hi Donato,

if you are using version 12 of the ospsuite R package, then the saved PKML can only be opened in v12 of MoBi, which you can download here: https://ci.appveyor.com/project/open-systems-pharmacology-ci/mobi/branch/develop/artifacts

This inconvenience will be solved as soon as we release v12 of OSPS (soon).

**Comment by teutonicod** [comment]
Hi Pavel,

Thanks a lot! Indeed I am using ospsuite v12! Sorry, I even forgot to mention the version I was using.
Thanks again for your help!

Donato
